RFC-001-BAM.md

```diff
@@ -50,6 +50,7 @@ This RFC describes BAM!: **B**i-directional **A**pplication **M**essaging. It is
 RFC-Number: 001
 Status: Draft
 ```
+Please feedback comments in [Tracking issue for RFC-001-BAM](../../issues/17).
```

Comment:
Hmm, I actually thought of something like this:

RFC-Number: 001
Status: Draft
Tracking-Issue: https://github.com/comit-network/RFCs/issues/17

Note that GitHub will create redirects automatically if we ever move/rename the repo again, so I think there is not need for a relative link here. Might actually be bad if we move around the files at some point.

Thoughts @comit-network/rust-devs?

Reply:
Why would we move the files in a repo different from the issues?

Reply:
Also, I don't think "Tracking-Issue" is enough to designate the issue. I personally had no clue of what were the issues created by @LLFourn until it was clarified.

Reply:
It is pretty common to have something like that. For example, see here for how they do it in rust-lang:

![image](https://user-images.githubusercontent.com/5486389/51508205-65729f00-1e48-11e9-9b29-ecad5dcfc578.png)

IMO, having this same sentence everywhere is too much noise and as soon as we have #24 done, it should be clear to what "Tracking issue"/"Discussion issue" refers to.

Reply:
I'll have to remove the code block formatting

Reply:
Not a big deal IMO.

Maybe we should also just move it to the top of the document?